subsys/bluetooth/controller/ll_sw/ull_llcp.c

```diff
@@ -219,6 +222,7 @@ struct proc_ctx *ull_cp_priv_create_local_procedure(enum llcp_proc proc)
 	default:
 		/* Unknown procedure */
 		LL_ASSERT(0);
+		break;
```

Comment:
Is this break due to personal preference or coding guide lines?
Asking as we are not doing this elsewhere, so perhaps we should?

Reply:
Misra.
https://gitlab.com/MISRA/MISRA-C/MISRA-C-2012/Example-Suite/-/blob/master/R_16_03.c